### PR TITLE
adds apply to db step

### PR DESCRIPTION
**Pull request by robert-ervin-jones**
- Adds apply to database step in Detailed Workflow section

### CODE REVIEW

File: pages/detailed-example-workflow/Step_5_apply_to_db.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Apply to Database
+parent: Detailed Example Workflow
+nav_order: 4
```
Comment (atice) [inline pages/detailed-example-workflow/Step_5_apply_to_db.md:5]: Should this really be 4? I don't think so.

File: pages/detailed-example-workflow/Step_5_apply_to_db.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - `-h`, `--help` Show this help message and exit.
+
+`apply_to_db.py` will first backup the current database/ directory and place the backup in `PhyloFisherDatabase_v1.0/database/backups`. Next, `apply_to_db.py` will append orthologs from input taxa to their corresponding gene files located in `PhyloFisherDatabase_v1.0/database/orthologs/`, append paralogs for input taxa to their corresponding gene files located in `PhyloFisherDatabase_v1.0/database/paralogs/`, and remove sequences marked for deletion. `apply_to_db.py` will update the binary matrix file `db_ortholog_occupancy.tsv` for newly added taxa based on decisions made during manual curation. The file `metadata.tsv` will also be updated with information provided for newly added taxa in `input_metadata.tsv`. Next, `apply_to_db.py` will rebuild the database so sequences from newly added taxa are present in the profile HMMs, the diamond database. This will allow orthologs from the newly added taxa to be used as specific queries in subsequent runs of addition. Finally, `apply_to_db.py` will copy the input proteome of all new taxa that are added to the database to `PhyloFisherDatabase_v1.0/database/proteomes/`.
```
Comment (atice) [inline pages/detailed-example-workflow/Step_5_apply_to_db.md:27]: Does the file db_ortholog_occupancy.tsv still exist? Maybe so and this is what explore database uses to get the number of orthologs present for a taxon?

**Review by atice — CHANGES_REQUESTED** [review]
A couple comments to double check.

File: pages/detailed-example-workflow/Step_5_apply_to_db.md (hunk 1 above)
Comment (robert-ervin-jones) [reply pages/detailed-example-workflow/Step_5_apply_to_db.md:5]: You're right. Should be 5

File: pages/detailed-example-workflow/Step_5_apply_to_db.md (hunk 2 above)
Comment (atice) [reply pages/detailed-example-workflow/Step_5_apply_to_db.md:27]: Yes that is fine.

**Review by atice — APPROVED** [review]
Looks good.

**Pull request merged**